lib/elliptic/eddsa/key.js

```diff
@@ -17,11 +17,18 @@ var cachedProperty = utils.cachedProperty;
 */
 function KeyPair(eddsa, params) {
   this.eddsa = eddsa;
-  this._secret = parseBytes(params.secret);
+  if (params.hasOwnProperty('secret'))
+    this._secret = parseBytes(params.secret);
   if (eddsa.isPoint(params.pub))
     this._pub = params.pub;
-  else
+  else {
     this._pubBytes = parseBytes(params.pub);
+    if (this._pubBytes && this._pubBytes.length === 33 &&
+        this._pubBytes[0] === 0x40)
```

Comment:
Please add `{` `}`.

Reply:
Only here?

Reply:
Yeah, current code style allows omitting braces if the `if` statement and its body fits into two lines total (the body must always be on the next line).